var/spack/repos/builtin/packages/ppopen-appl-amr-fdm/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class PpopenApplAmrFdm(MakefilePackage):
+    """
+    ppOpen-APPL/AMR-FDM is an adaptive mesh refinement (AMR) framework
+    for development of 3D parallel finite-difference method (FDM)
+    applications by the following capabilities:
+
+    - Cell-based adaptive mesh structure
+    - Dynamic domain decomposition (DDD) to avoid load imbalance problem
+    - Explicit time integration
+    - Flat MPI & OpenMP/MPI hybrid parallel programming models
```

Comment:
The formatting of this section will break when you run `spack info ppopen-appl-amr-fdm`. I would either remove these details, or add commas or something.

Reply:
fixed.